core/trino-spi/src/test/java/io/trino/spi/type/TestSqlVarbinary.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.type;
+
+import org.testng.annotations.Test;
```

Comment:
Use junit

Reply:
Currently `trino-spi` doesn't have junit on the classpath for testing dependencies and all the other tests are using `org.testng.annotations.Test`. Are we trying to switch the preferred annotations for new PR's over to junit instead?

Reply:
We’re slowly trying to migrate to JUnit (https://github.com/trinodb/trino/issues/9378) so I think we should try to take the opportunity to use it when adding new tests. Otherwise, it’s more work someone has to do later.

Reply:
Got it- good to know. I tried switching just this one test to use jUnit, but it seemed like adding junit to `trino-spi/pom.xml' prevented all of the existing testng tests from running. A quick google search seemed to expose ways to run jUnit tests within TestNG, but I couldn't find anything about doing it the other way around- so it seems like using jUnit here would require migrating all of the `trino-spi` tests all at once, unless you know of a way to avoid it.

Reply:
Nevermind, I see that `trino-main/pom.xml` has a configuration for allowing both to co-exist. I'll push a new commit in this PR to add jUnit support to `trino-spi` and migrate this new test to use those annotations instead.